packages/paste-website/src/pages/components/avatar/index.mdx

```diff
@@ -138,6 +143,11 @@ Provide the Avatar with a source path via the `src` prop to render the Avatar as
 
 Provide the Avatar with an `icon` prop to display an icon. You must import the icon before passing it to the `icon` prop. You **must** still provide a `name` prop. The icon must be a <a href="https://paste.twilio.design/icons/">Paste Icon</a>.
 
+<Callout>
```

Comment:
I think with the new typings we can probably remove these callouts now. Previously we added a note because the behavior was allowed and possibly unexpected, but now the code will throw an error informing the developer.

Reply:
That's a fair observation. Is it worth keeping for those who don't use TS and ignore propTypes?

I'm easy, either way

Reply:
My instinct is to leave them there since I can't think of any downsides to having repeated information - what do you think? It seems like it could be more beneficial for an engineer to notice the rule in the docs and avoid having to encounter the error at all, rather than debugging the error.